### PR TITLE
Add a rpm repo refresh step ahead of the rpm sync

### PR DESCRIPTION
**Pull request by jdotpz**

### CODE REVIEW

File: bin/deploylib/create_rpm.sh [hunk 1]
```diff
@@ -29,6 +29,9 @@ function create_rpm() {
     echo "`date` -- Signed socorro package, now copying into local repo"
     # Copy to the local repo
     cp ${NEWRPM} ~/org.mozilla.crash-stats.packages-public/x86_64
+    echo "`date` -- Refreshing RPM repo from S3"
```
Comment (phrawzty) [inline bin/deploylib/create_rpm.sh]: `refresh` will wipe out local changes - including, notably, the file copied in the line above. :boom:

Suggested workflow:
* Create RPM
* Sign RPM
* Refresh repo
* Copy RPM into position
* Update repo

**Comment by jdotpz** [comment]
Excellent spot, thanks @phrawzty !
Should be ready to rock.

File: bin/deploylib/create_rpm.sh [hunk 2]
```diff
@@ -26,6 +26,8 @@ function create_rpm() {
     SOCORROAMINAME="${NEWSOCORROVERSION}-`date +%m%d%y`"
     # Sign the rpm file
     rpm --addsign ${NEWRPM} < /home/centos/.rpmsign
+    echo "`date` -- Refreshing RPM repo from S3"
+    /home/centos/manage_repo.sh refresh
     echo "`date` -- Signed socorro package, now copying into local repo"
```
Comment (phrawzty) [inline bin/deploylib/create_rpm.sh]: Nit, but this log message should be moved up under the actual signing step. :)

**Comment by phrawzty** [comment]
`r+` with nit.

**Pull request merged**